next.config.js

```diff
@@ -5,29 +5,29 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 const { NODE_ENV, CDN_FRONTEND_BASE_URL } = process.env
 
 module.exports = withBundleAnalyzer({
+  future: {
+    webpack5: true
+  },
   webpack: config => {
     config.externals = config.externals || {}
     config.externals['lru-cache'] = 'lru-cache'
     config.externals['react-dom/server'] = 'react-dom/server'
 
-    const entryFactory = config.entry
-    const polyfillPath = './lib/polyfill.js'
-
     const alias = Object.assign({}, config.resolve.alias)
     delete alias.url
     config.resolve = {
       ...config.resolve,
       alias
     }
 
-    config.entry = async () => {
```

Comment:
what did this do previously? just curious

Reply:
long time ago next.js didn't ship polyfills for old browsers and didn't have an `_app.js` that ran before everything else: therefore we had to customize webpack to include a polyfill before anything else ran